app/src/main/java/org/wikipedia/page/tabs/TabActivity.java

```diff
@@ -278,6 +285,16 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
+    private void saveTabsToList() {
```

Comment:
Would it be better to add the logic of checking "empty" tab when saving to list?

Reply:
good one will add..

Reply:
Done @cooltey

Reply:
Nice! Can we have a message if the "Save all tabs" action actually does nothing?

Reply:
have asked for a copy on ticket @cooltey .. if everything else is good we can merge and then add the copy as follow-up?

Reply:
Make sense!